### PR TITLE
Adding code coverage for Apex and LWC

### PR DESCRIPTION
**Pull request by muenzpraeger**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -50,7 +50,13 @@ jobs:
...
                   name: Unit test Lightning Web Components
                   command: |
-                      npm run test:unit -- -- --runInBand
+                      npm run test:unit:coverage -- -- --runInBand
+            - persist_to_workspace:
+                  root: ~/
+                  paths:
+                      - .sfdx/*
+                      - project/*
+                      - coverage/*
```
Comment (zaynelt) [inline .circleci/config.yml]: Can we add a comment here about what this does? (i.e. we're storing code coverage results)

File: .circleci/config.yml [hunk 2]
```diff
@@ -154,7 +160,28 @@ jobs:
             - run:
                   name: Run Apex Tests
                   command: |
-                      node_modules/sfdx-cli/bin/run force:apex:test:run -u ciorg -c -r human
+                      mkdir -p ~/tests/apex
+                      node_modules/sfdx-cli/bin/run force:apex:test:run -u ciorg -c -r human -d ~/tests/apex -w 20
+            - persist_to_workspace:
+                  root: ~/
+                  paths:
+                      - .sfdx/*
```
Comment (zaynelt) [inline .circleci/config.yml]: Samsies here. Apex results and maybe unpack the cli command '-r human', blah blah.

File: .circleci/config.yml (hunk 1 above)
Comment (muenzpraeger) [reply .circleci/config.yml]: Added

File: .circleci/config.yml (hunk 2 above)
Comment (muenzpraeger) [reply .circleci/config.yml]: Added

**Pull request merged**